tests/sles4sap/saptune/mr_test.pm

```diff
@@ -23,7 +23,7 @@ sub reboot_wait {
     $self->reboot;
 
     # Wait for tuned to tune everything
-    my $timeout = 60;
+    my $timeout = 60 * get_var('TIMEOUT_SCALE', 1);
```

Comment:
We're using `bmwqemu::scale_timeout(60)` instead of a explicit operation with `TIMEOUT_SCALE`.

Reply:
Fixed.